packages/framework/view-interfaces/src/mountableView.ts

```diff
@@ -49,3 +49,12 @@ export interface IFluidMountableView extends IProvideFluidMountableView {
 	 */
 	unmount(): void;
 }
+
+/**
+ * TODO: determine what is needed here
+ */
+export interface IFluidMountableViewEntryPoint {
```

Comment:
This should live next to ContainerViewRuntimeFactory rather than in view-interfaces.  This also keeps it off the public API surface.

Reply:
I did this initially, but `@fluid-tools/webpack-fluid-loader` doesn't have a dependency on `@fluid-example/example-utils`, so the typing wouldn't be completely safe.
Is it ok to make the webpack fluid loader code not be super type safe for this?